src/views/bibliography_element.ts

```diff
@@ -203,12 +199,12 @@ export class BibliographyElementBlockView<
   }
 
   private handleSave = (item: BibliographyItem) => {
-    const node = this.decoder.decode(item) as BibliographyItemNode
-    this.updateNodeAttrs(node.attrs)
+    const node = decode(item)
```

Comment:
I suppose in the final version there'd be no need to decode as we'll just receive the attributes, right?

Reply:
Right. I reverted most of my changes to this file, _but the decode is still there_ since it was there before. Once this PR is merged, I'll do the same for references/citations and remove the decode.